### PR TITLE
Support PVC

### PR DESCRIPTION
**Pull request by urihoenig**

### CODE REVIEW

File: stable/jupyter/templates/jupyter-deployment.yaml [hunk 1]
```diff
@@ -177,3 +180,6 @@ spec:
 {{- if .Values.volumes }}
 {{ include .Values.volumes.volumesTemplate . | indent 8 }}
 {{- end }}
+{{- if .Values.pvc }}
+{{ include .Values.pvc.pvcTemplate . | indent 8 }}
```
Comment (omesser) [inline stable/jupyter/templates/jupyter-deployment.yaml]:
```suggestion
{{ include .Values.pvc.pvcVolumesTemplate . | indent 8 }}
```
I suppose? to differentiate and keep the naming conventions established

File: stable/jupyter/values.yaml [hunk 2]
```diff
@@ -82,6 +82,10 @@ volumes:
   volumesTemplate: v3io-configs.deployment.mount-with-fuse-and-home
   volumeMountsTemplate: v3io-configs.deployment.volumeMounts-with-fuse-and-home
 
+pvc:
+  pvcTemplate: pvc.volumes
```
Comment (omesser) [inline stable/jupyter/values.yaml]:
```suggestion
  pvcVolumesTemplate: pvc.volumes
```

File: stable/pvc/templates/_pvc.tpl [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{{- define "pvc.volumes" -}}
```
Comment (omesser) [inline stable/pvc/templates/_pvc.tpl]: should the `pvc` level be here?
I'm not sure, I just see `pvc: ...` is in jupyter and pvc helm chart has no values of it's own

**Review by omesser — DISMISSED** [review]

File: stable/jupyter/templates/jupyter-deployment.yaml (hunk 1 above)
Comment (urihoenig) [reply stable/jupyter/templates/jupyter-deployment.yaml]: ok

File: stable/pvc/templates/_pvc.tpl (hunk 3 above)
Comment (urihoenig) [reply stable/pvc/templates/_pvc.tpl]: it's the value of pvc.pvcVolumesTemplate. 
It's the same as in v3io-configs

**Pull request merged**